Added synchronous counter interactive example

fluctix opened the pull request:
Solves #116 
[GCI Task](https://codein.withgoogle.com/dashboard/task-instances/4677084887121920/)

https://circuitverse.org/simulator/embed/43082

Screenshots:
![embedSCshot](https://user-images.githubusercontent.com/40538506/70846678-cdc3d800-1e81-11ea-8f00-4702e3c4b35f.PNG)

fluctix commented:
@sal2701 @PragatiVerma18 @sakshi1499 Please review.

Ananthavijay commented:
@fluctix I think you can also fix the broken image under Synchronous counters ( At last )

sakshi1499 commented:
@fluctix It looks good to me. @Ananthavijay which image are you talking about can you attach a screenshot? I can't see any broken image in my fork

sakshi1499 commented:
@PragatiVerma18 @sal2701 please look at it once?

Ananthavijay commented:
@sakshi1499 This is the one I am talking about.

![Screenshot (103)](https://user-images.githubusercontent.com/35162705/70853258-7baaa300-1ed1-11ea-85ac-c747846bccfb.png)

sakshi1499 commented:
![Screenshot_20191215-002814](https://user-images.githubusercontent.com/42520146/70853285-e065fd80-1ed1-11ea-8bc8-d393911bbf51.png)


See its fine in mine!

Ananthavijay commented:
@sakshi1499 check it here : https://github.com/CircuitVerse/Interactive-Book/blob/master/docs/counter.md

sakshi1499 commented:
Yes I have seen. The image is broken on Github .md page, its working fine on the website. Please edit that too @fluctix

sal2701 commented:
@sakshi1499 I think we should show 2 outputs, **Qa** and **Qb** as it'll better demonstrate the 2 Bit Counter. A single output here is confusing. What say?

fluctix commented:
Updated the Markdown file to point to the correct location of the Image(as requested by @Ananthavijay ) and also as @sal2701 suggested, I renamed the circuit from 'main' to 'Synchronous Counter' , and added one more output(ie. Qa) for better understanding.
Screenshots:
![Screenshot1](https://user-images.githubusercontent.com/40538506/70856902-b54bd000-1f0a-11ea-905e-341eb09a4ee8.PNG)
![Screenshot2](https://user-images.githubusercontent.com/40538506/70856903-b54bd000-1f0a-11ea-92ce-88dcf088914c.PNG)

fluctix commented:
@sakshi1499 @sal2701 Please review.

PragatiVerma18 commented:
Looks good to me!! Great work @fluctix 😃

sakshi1499 commented:
Great work!. @sal2701 that was a good suggestion.

sakshi1499 commented:
@sal2701 Please look at it if you find no issues please approve it?

fluctix commented:
New changes have brought merge conflicts....

Shivansh2407 commented:
@criticic can you please fix the merge conflicts?
